documentation/docs/verifiable_credentials/create.mdx

```diff
@@ -12,15 +12,67 @@ keywords:
+
+- **Created**: timestamp of when the credential was signed.
+- **Expires**: timestamp after which the credential is no longer considered valid.
+- **Challenge**: arbitrary random string. Sent by the issuer and mitigates replay attacks; should be sufficiently random and uniquely generated per credential request.
+- **Domain**: arbitrary string. Sent by the issuer and can help mitigate replay attacks when used with a challenge.
+- **Proof Purpose**: indicates the purpose of the signature.
+  - **AssertionMethod**: to assert a claim. The signing verification method must have an [`assertionMethod`](https://www.w3.org/TR/did-core/#assertion) relationship to be valid.
+  - **Authentication**: to authenticate the signer. The signing verification method must have an [`authentication`](https://www.w3.org/TR/did-core/#authentication) relationship to be valid.
+
+### Credential Validation {#credential-validation}
 
-This example shows how you can create a Verifiable Credential and validate it. In this example, Alice takes the role of the subject, while we also have an issuer.
-The issuer signs a UniversityDegreeCredential type verifiable credential with Alice's name and DID.
-This Verifiable Credential can be verified by anyone, allowing Alice to take control of it and share it with anyone.
+Both issuers and holders of verifiable credentials might be interested in making sure certain properties of the VC are valid, specially right after creating or receiving a VC, but this checks can be done at any point in time and can be a useful way of making sure the VC has not been revoked. 
+
+The Iota Framework allows issuers and holders to perform some validations using the CredentialValidator.
+
+The following checks can be verified by the validator:
+
+- **Credential proofs**: verifies the credential signatures against the DID Documents of the issuer.
+- **Semantic Validation**: ensures the credential adhere to the specification.
+- **Subject Holder Relationship**: ensures that the credential relationship between the holder and the subject match.
+- **Optional validations**: additional checks on signatures and credential fields can be configured.
+- **Latest Issuance date**: check that the [`issuanceDate`](https://www.w3.org/TR/vc-data-model/#issuance-date) property, if present, is not after a specific datetime. Defaults to the current datetime if unset.
+- **Earliest Expiry date**: check that the [`expirationDate`](https://www.w3.org/TR/vc-data-model/#expiration) property, if present, is not before a specific datetime. Defaults to the current datetime if unset.
+
+The last two checks listed above are specified using CredentialValidationOptions.
+
+### Sharing Verifiable Credentials {#sharing-verifiable-credentials}
+
+A verifiable presentation is the recommended data format for sharing one or more verifiable credentials. The next section explains [verifiable presentations](./verifiable_presentations.mdx) in more detail.
```

Comment:
I would avoid mentioning the order of the articles/pages, since `Revocation` is actually the next page in the sidebar.

```suggestion
### Sharing Verifiable Credentials {#sharing-verifiable-credentials}

A [verifiable presentation](./verifiable_presentations.md) is the recommended data format for sharing one or more verifiable credentials, as it provides cryptographic means of proving the DID of the holder presenting them, and for enforcing [subject-holder relationships](https://www.w3.org/TR/vc-data-model/#subject-holder-relationships). See the [Verifiable Presentations](./verifiable_presentations.md) page for further detail.
```